Example/PrebidDemo/PrebidDemoObjectiveC/Examples/InApp/CustomRendererDisplayBannerViewController.m

```diff
@@ -0,0 +1,62 @@
+/*   Copyright 2019-2022 Prebid.org, Inc.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "CustomRendererDisplayBannerViewController.h"
+#import "PrebidDemoMacros.h"
+
+NSString * const storedImpDisplayBannerCustomRenderer = @"prebid-demo-banner-320-50";
```

Comment:
What is the purpose of these files? 

From the GitHub UI I don't see changes in the project. 

And as I understand, the integration case for third-party rendering should use `prebid-ita-banner-320-50-meta-custom-renderer` for proper work. 

It also should be an integration case for the Swift demo app.